sdks/python/apache_beam/transforms/core.py

```diff
@@ -2507,38 +2507,32 @@ def get_output_type(self):
 
   def expand(self, pbegin):
     assert isinstance(pbegin, pvalue.PBegin)
-    # Must guard against this as some legacy runners don't implement impulse.
-    debug_options = pbegin.pipeline._options.view_as(DebugOptions)
-    fn_api = (debug_options.experiments
-              and 'beam_fn_api' in debug_options.experiments)
-    if fn_api:
```

Comment:
I'm assuming things like this can be dropped since all runners other than Dataflow are already on portability?

Reply:
Yep.

Reply:
(Well, the old bundle based direct runner isn't, but as of this PR it can still handle this construction.)

Reply:
Well, the old bundle base direct runner isn't, but as of this PR can support this construction.